xcodeproj/internal/bazel_integration_files/clang.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -euo pipefail
+
+while test $# -gt 0
+do
+  case $1 in
+  -MF)
+    shift
+    touch "$1"
+    ;;
+  --serialize-diagnostics)
+    shift
+    cp "${BASH_SOURCE%/*}/cc.dia" "$1"
+    ;;
+  *.o)
+    break
+    ;;
+  -v)
+    clang=$(xcrun --find clang)
```

Comment:
TODO: Check if the developer directory is in the environment so that we can avoid this call.